azurecompute-arm/src/test/java/org/jclouds/azurecompute/arm/features/VirtualMachineScaleSetApiMockTest.java

```diff
@@ -0,0 +1,488 @@
+              vmssname,
+              "eastus",
+              VirtualMachineScaleSetSKU.create(
+                      "Standard_A1",
+                      "Standard",
+                      10),
+              null,
+              VirtualMachineScaleSetProperties.create(
+                      true,
+                      true,
+                      VirtualMachineScaleSetUpgradePolicy.create("Manual"),
+                      VirtualMachineScaleSetVirtualMachineProfile.create(
+                              StorageProfile.create(
+                                      ImageReference.create(
+                                              null,
+                                              "Canonical",
+                                              "UbuntuServer",
+                                              "16.04-LTS",
+                                              "latest"),
+                                      OSDisk.create(
+                                              "Windows",
+                                              null,
+                                              null,
+                                              null,
+                                              "FromImage",
+                                              null,
+                                              ManagedDiskParameters.create(
+                                                      null,
+                                                      "Standard_LRS"),
+                                              null),
+                                      Arrays.asList(DataDisk.create(
+                                              null,
+                                              "10",
+                                              1,
+                                              null,
+                                              null,
+                                              "FromImage",
+                                              "None",
+                                              ManagedDiskParameters.create(
+                                                      null,
+                                                      "Standard_LRS"),
+                                              null))),
+                              VirtualMachineScaleSetOSProfile.create(
+                                      "jclouds-vmssname",
+                                      "jclouds",
+                                      "jClouds1!",
+                                      VirtualMachineScaleSetOSProfile.LinuxConfiguration.create(
+                                              "False",
+                                              null),
+                                      null,
+                                      new ArrayList<Secrets>()),
+                              getNetworkProfile(),
+                              getExtensionProfile()
+
+                      )
+              ));
+   }
```

Comment:
Same here. Use just one "testCreate" complete enough to validate all properties you want to validate and remove all the redundant ones.